test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.test.cpp

```diff
@@ -763,6 +763,27 @@ TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit, GetPassWithRevolutionNumbe
             );
         }
     }
+
+    {
+        // Environment setup
+
+        const Environment environment = Environment::Default();
+
+        // Orbit setup
+
+        const TLE tle = {
+            "1  7276U 74026A   19177.06815612 -.00000040  00000-0 -59146-3 0  9999",
+            "2  7276  63.5942  41.6062 6850823 289.3685  12.0680  2.45095050222705",
+        };
+
+        const SGP4 sgp4 = SGP4(tle);
+
+        const Orbit orbit = {sgp4, environment.accessCelestialObjectWithName("Earth")};
+
+        const Pass pass = orbit.getPassWithRevolutionNumber(22700, Duration::Minutes(30.0));
+
+        EXPECT_TRUE(pass.isDefined());
```

Comment:
In general I would agree, but this is a regression test to ensure that a pass was calculated, not a test to measure the accuracy of the pass. That would require generating all the passes in STK or equivalent tool which is overkill considering we have atleast 3 tests above that do exactly that.
In my opinion a waterfall of tests where you validate individual methods and then add scenario based tests above is more that sufficient to test all the use cases.

Reply:
This can just be a self-validation sort of test, where you assert something like this to "lock" this output pass # 22273 as having occured at that time
```suggestion
        EXPECT_TRUE(pass.isDefined());
        EXPECT_EQ(pass.accessInstantAtAscendingNode(),Instant::DateTime("corresponding instant"));
```